src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -543,7 +546,10 @@ public virtual async Task BreakCondition(string bkptno, string expr)
 
         public virtual IEnumerable<Guid> GetSupportedExceptionCategories()
         {
-            return new Guid[0];
+            // return new Guid[0];
+            List<Guid> categories = new List<Guid>();
+            categories.Add(new Guid("{3A12D0B7-C26C-11D0-B442-00A0244A1DD2}")); // maybe there is a better way than this...
```

Comment:
> // maybe there is a better way than this... [](http://example.com/codeflow?start=79&length=44)

```
const string CppExceptionCategoryString = "{3A12D0B7-C26C-11D0-B442-00A0244A1DD2}";

return new Guid[] { new Guid(CppExceptionCategoryString) };
```

Reply:
@gregg-miskelly Do we need to add `ExceptionSettings` to [cppdbg.ad7Engine.json](https://github.com/microsoft/MIEngine/blob/main/src/OpenDebugAD7/cppdbg.ad7Engine.json)?

E.g.
```
"exceptionSettings": {
   "categories": [
      {
         "name": "CppExceptionCategory",
         "id": "3A12D0B7-C26C-11D0-B442-00A0244A1DD2"
      }
   ]
}
```

or should we change the code to ignore it for VS Code for now since we wont have the DAP side of it implemented for this feature?
https://github.com/microsoft/MIEngine/blob/8456c49b4731971b885c2909f9feaac4bdcd494d/src/DebugEngineHost.VSCode/HostConfigurationStore.cs#L55-L65

Reply:
Got it. Yes, it sounds like we need to add the place holder value that you have above.